sensor/src/constants.h

```diff
@@ -1,9 +1,9 @@
 #define COMMAND_UP          "fb0603450e0001fe66"
 #define COMMAND_DOWN        "fb0603450e0002fd50"
 //#define COMMAND_TOGGLE      "fb066666666609f6c2"
-// #define COMMAND_TIME        "fb066666666603fc76"
+#define COMMAND_TIME        "fb0664d4060003fc76"
 #define COMMAND_CHANGE_MODE "fb0603450e0004fb3c"
-// #define COMMAND_BLOWER      "fb06666666660af5f4"
+#define COMMAND_AUX         "fb06666666660af5f4" // TODO: FIXME - Not actual value
```

Comment:
Do you have a valid FB for this @tmjo ?

Reply:
Hi, I just checked. The FB - in my tub at least - is the same as for Pump2 (which I don't have, or which is my Aux). Since mine is not working (but I can toggle it on and off on the panel) I have obviously not paid enough attention to it.

Anyways, commands are same as we have for Pump2. What I discovered when testing it today is that the status for Aux is not the same as for Pump2. It is in fact in the character before:

```
fa143339304300006100000080020e0c66000000000039 --> Aux off
fa1433393043c0006100000480020e0d6600000000008e --> Aux on (notice c after our 43/degC)
```

I made a small hack and then the HA pump2 selector was working, but perhaps it is better to add a separate one.

As you see there is also a change in the unknown bytes going from 8002 to 48002. This was repeated a few times, but I don't know what it means. Perhaps it could be an indicator of what aux is actually running? Dunno.